billy_metadata/sd.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+
+
+settings = dict(
```

Comment:
I am not sure about this one actually, I remember checking your mi PR for pupa conversion, for this purpose, you seemed to have avoided settings dictionary, though I am not sure if I can avoid it for SD.